orchestrator-salt/src/main/resources/salt/salt/sssd/template/generate_cm_keytab.j2

```diff
@@ -7,15 +7,27 @@ echo "$password" | kinit {{ ipa.principal }}
 
 set -x
 
+SA_NAME=krbbind-{{ ipa.cluster_name }}
+ROLE_NAME=hadoopadminrole-{{ ipa.cluster_name }}
+
+# Generate per-cluster krbbind Service Account
+ipa user-add "$SA_NAME" --first=service --last=account
```

Comment:
user and role mgmt final place should be in the freeipa mgmt service

Reply:
then we need a flow to tell the IPA mgmt service to create / delete these roles. we can discuss that scenario, until then this can be used